test/integration-test-v2.py

```diff
@@ -38,6 +39,9 @@ def main():
 def test_multidomain():
     auth_and_issue([random_domain(), random_domain()])
 
+def test_wildcardmultidomain():
+    auth_and_issue([random_domain(), "*."+random_domain()], chall_type="dns-01")
```

Comment:
This is a good test case; let's add another one that specifically for a random domain, plus the wildcard version of that same random domain, which is likely to be a common case. And if it's easy, check the resulting order object to make sure it has two authorizations.

Reply:
Sure :+1: